src/styles/globals.css

```diff
@@ -168,4 +195,80 @@
     display: flex;
     justify-content: center;
     border: solid black 3px;
+    border-bottom: 0;
+}
+
+.wrapper {
+    display: flex;
+    flex-direction: column;
+}
+
+.navbar {
+    display: flex;
+    flex-direction: column;
+}
+
+.main {
+    display: flex;
+    justify-content: center;
+}
+
+.bodywrapper {
+    display: flex;
+    height: calc(100vh - 100px);
+    flex-direction: row-reverse;
+}
+
+.footer-pos {
+    display: flex;
+    justify-content: center;
+}
+
+.body {
+    width: 100%;
+    height: calc(100vh - 100px);
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    flex-direction: column;
+}
+
+.pokeball {
+    display: flex;
+    align-items: flex-start;
+    height: 100%;
+    background: url(/pixelpokeball.png) 0 0;
+    aspect-ratio: 75 / 100;
+}
+
+.pokeball:hover {
+    /* background: url(/pixelpokeball.png) -148px 0; */
+    animation-name: pokeballopening;
+    animation-duration: 1s;
+    animation-fill-mode: forwards;
+}
+
+.themebutton {
+    margin: 10px;
+}
+
+@keyframes pokeballopening {
+    0% {
+        background: url(/pixelpokeball.png) 0 0;
```

Comment:
Da die URL des Backgrounds immer gleich bleibt, brauchst du sie nicht in jedem Frame mit angeben. In Keyframes sollen grundsätzlich nur Props gelistet werden, die sich im Laufe der Animation ändern.

`background` ist eine übergeordnete CSS Prop, die sich hier bei dir zusammensetzt aus ´background-image` und `background-position`.

Bei dir ändert sich während der Animation ja aber nur die Position.